### PR TITLE
Established and exposed ComponentImage offset

### PR DESCRIPTION
**Pull request by devildrake**

### CODE REVIEW

File: Project/Source/Components/UI/ComponentImage.cpp [hunk 1]
```diff
@@ -233,6 +245,14 @@ void ComponentImage::SetAlphaTransparency(bool alphaTransparency_) {
 	alphaTransparency = alphaTransparency_;
 }
 
+TESSERACT_ENGINE_API float2 ComponentImage::GetTextureOffset() const {
+	return textureOffset;
+}
+
+TESSERACT_ENGINE_API void ComponentImage::SetTextureOffset(float2 textureOffset_) {
```
Comment (magalenyo) [inline Project/Source/Components/UI/ComponentImage.cpp:252]: You don't really need the TESSERACT_ENGINE_API notation here (just in the .h). I suppose this has been tested in a Gameplay script and works correctly with the notation.

**Review by magalenyo — APPROVED** [review]

**Review by JaimeBea — DISMISSED** [review]

File: Project/Source/Components/UI/ComponentImage.cpp (hunk 1 above)
Comment (JaimeBea) [reply Project/Source/Components/UI/ComponentImage.cpp:252]: Correct, this isn't necessary

**Review by magalenyo — APPROVED** [review]

**Review by JaimeBea — APPROVED** [review]
Check what Miguel said in the comment above.

File: Project/Source/Components/UI/ComponentImage.cpp (hunk 1 above)
Comment (devildrake) [reply Project/Source/Components/UI/ComponentImage.cpp]: Copy that

**Review by JaimeBea — APPROVED** [review]

**Review by lowysole — APPROVED** [review]

**Review by magalenyo — APPROVED** [review]

**Pull request merged**